Content.Server/Nutrition/EntitySystems/DrinkSystem.cs

```diff
@@ -123,51 +126,42 @@ public float TotalHydration(EntityUid uid, DrinkComponent? comp = null)
 
     private void OnExamined(EntityUid uid, DrinkComponent component, ExaminedEvent args)
     {
-        if (!component.Opened || !args.IsInDetailsRange || !component.Examinable)
+        var canOpen = TryComp<OpenableComponent>(uid, out var openable);
+        if (_openable.IsClosed(uid, openable) || !args.IsInDetailsRange || !component.Examinable)
             return;
 
-        var color = IsEmpty(uid, component) ? "gray" : "yellow";
-        var openedText =
-            Loc.GetString(IsEmpty(uid, component) ? "drink-component-on-examine-is-empty" : "drink-component-on-examine-is-opened");
-        args.Message.AddMarkup($"\n{Loc.GetString("drink-component-on-examine-details-text", ("colorName", color), ("text", openedText))}");
-        if (!IsEmpty(uid, component))
+        var empty = IsEmpty(uid, component);
+        var color = empty ? "gray" : "yellow";
+        // only show the Opened text if it can be opened
+        if (empty || canOpen)
         {
-            if (TryComp<ExaminableSolutionComponent>(uid, out var comp))
-            {
-                //provide exact measurement for beakers
-                args.Message.AddMarkup($" - {Loc.GetString("drink-component-on-examine-exact-volume", ("amount", DrinkVolume(uid, component)))}");
-            }
-            else
-            {
-                //general approximation
-                var remainingString = (int) _solutionContainer.PercentFull(uid) switch
-                {
-                    100 => "drink-component-on-examine-is-full",
-                    > 66 => "drink-component-on-examine-is-mostly-full",
-                    > 33 => HalfEmptyOrHalfFull(args),
-                    _ => "drink-component-on-examine-is-mostly-empty",
-                };
-                args.Message.AddMarkup($" - {Loc.GetString(remainingString)}");
-            }
+            var openedText =
+                Loc.GetString(empty ? "drink-component-on-examine-is-empty" : "drink-component-on-examine-is-opened");
+            args.Message.AddMarkup($"\n{Loc.GetString("drink-component-on-examine-details-text", ("colorName", color), ("text", openedText))}");
```

Comment:
shoudnt this be handled by openable? (the empty part should already be handled by ExaminableSolution?)

Reply:
bananas shouldnt have a description saying opened, only drinks

Reply:
> bananas shouldnt have a description saying opened, only drinks

No, but they *should* have one saying "peeled" shrimply make the message a datafield

Reply:
yes then

Reply:
dew it

Reply:
i did